syft/frameworks/torch/tensors/interpreters/large_precision.py

```diff
@@ -61,30 +60,38 @@ def _create_internal_tensor(self):
         result = np.array(result).reshape(new_shape)
         return torch.tensor(result, dtype=self.internal_type)
 
+    def _create_tensor_from_numpy(self, ndarray):
```

Comment:
You can make this method a @staticmethod like _internal_representation_to_large_ints, and change the signature `_create_tensor_from_numpy(self, ndarray):` -> `_create_tensor_from_numpy(ndarray, internal_type):`. I believe you can get `self.child.shape[:-1] ` for the ndarray directly (am i right?)